references/video_classification/train.py

```diff
@@ -95,6 +108,11 @@ def evaluate(model, criterion, data_loader, device):
             top1=metric_logger.acc1, top5=metric_logger.acc5
```

Comment:
@datumbox For the print I follow the format like this one. And the original print is saying `Clip Acc@1 ...` and for the new one I change `Clip` into `Video` below. However this might be quite confusing. Do you have good idea for the accuracy name here?

Reply:
I agree that printing 2 stats could be confusing. The thing is, even the "Video" one is clips; they are just aggregated all together instead on per video basis. One way around this would be to make your changes directly on the metric_logger but this would require additional checks. For now I think it's OK to show both; this can be reviewed on follow up PRs. Concerning the naming, the technique seems to be doing an ensemble of the clips, so you could name it as such. I would definitely not call it Clip vs Video because they are both based on clips.